kratos/python/add_utilities_to_python.cpp

```diff
@@ -420,6 +423,26 @@ void AddUtilitiesToPython(pybind11::module& m)
     .def("TestGetExactIntegration",&ExactMortarIntegrationUtility<3,4>::TestGetExactIntegration)
     .def("TestGetExactAreaIntegration",&ExactMortarIntegrationUtility<3,4>::TestGetExactAreaIntegration)
     ;
+
+    // Process Factory utility
+    class_<ProcessFactoryUtility, typename ProcessFactoryUtility::Pointer>(m, "ProcessFactoryUtility")
```

Comment:
I don't like the idea of calling python processes from c++.

still this is something that we could discuss, as a part of a different PR in any case.

Reply:
This is already on the ContactStructuralMechanicsApplication, we can discuss